db/view/view.cc

```diff
@@ -1095,9 +1095,13 @@ future<> mutate_MV(
                 cf_stats.total_view_updates_failed_local += is_local;
                 cf_stats.total_view_updates_failed_remote += remotes;
                 auto ep = f.get_exception();
+                tracing::trace(tr_state, "Failed to apply {}view update for {} and {} remote endpoints",
+                        is_local ? "local " : "", target, remotes);
```

Comment:
This drowns in the noise, but seastar::value_of() will avoid the is_local check when tracing is disabled.

Reply:
done